main.js

```diff
@@ -200,6 +208,19 @@ export class IdEncoder {
    * @returns {string} - Encoded string.
    */
   encode(bytes) {
+    if(this.multihash) {
+      const BYTE_SIZE = bytes.length;
+      // <varint hash fn code> <varint digest size in bytes> <hash fn output>
+      //  <identity function>              <32>                <seed bytes>
+      const seedMultihash = new Uint8Array(2 + BYTE_SIZE);
```

Comment:
This is not specific to "seeds" so none of the variables here should use that terminology. Using multihash is valuable because it enables an additional check to ensure that the expected size of the encoded data matches what is (later) decoded. This is true for any bytes that are encoded -- whether those bytes are meant to be for a secret key seed or just as a random identifier (but note that these two things should never be confused with one another as mentioned in other comments).

Reply:
Updated 155dcae